ExporterTest/ExporterTest.vcxproj

```diff
@@ -30,7 +30,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v143</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>

```

Comment:
Was the change in platform toolset intentional?

Reply:
I didn't intend on PRing it but since almost nobody uses MSVC for building anyway (and if they did they would probably have 2022) I think its fine.

Reply:
Fair enough.